src/lib/lite_base/nat.ml

```diff
@@ -31,10 +43,27 @@ module Make (M : Input_intf) = struct
           go init t 0 ) }
 
   let fold t = Fold_lib.Fold.group3 ~default:false (fold t)
+
+  module Importable = struct
+    type t = Stable.Latest.t [@@deriving eq, sexp, compare]
+
+    let length_in_triples, fold = (length_in_triples, fold)
+  end
 end
 
-module Make32 () = Make (struct
-  type t = Core_kernel.Int32.t [@@deriving bin_io, eq, sexp, compare]
+module Input_32_V1 = struct
```

Comment:
see comment above .... I think this should just be `Input_32`, which can contain all possible versions.